src/layers/SchematicLayer.ts

```diff
@@ -375,6 +432,28 @@ export class SchematicLayer<T extends SchematicData> extends PixiLayer<T> {
           this.drawCementPlug(obj, casings, holeSizes);
         }
       });
+
+    this.internalLayerVisibility.pAndALayerId &&
+      remainingPAndA.forEach((obj) => {
+        if (isPAndASymbol(obj)) {
+          const symbolRenderObject = this.prepareSymbolRenderObject(obj);
+          this.drawSymbolComponent(symbolRenderObject);
+        }
+        if (isCementPlug(obj)) {
+          this.drawCementPlug(obj, casings, holeSizes);
+        }
+      });
```

Comment:
Can be removed as the functionality is covered from L425-434
Probably addition from the rebase 😅

Reply:
I need a re-review on this one. I completely lost track of it in the refactor...